sciserver_catalogs/environment.yml

```diff
@@ -7,14 +7,13 @@ dependencies:
   - distributed
   - bottleneck
   - netCDF4
-  - xarray>=0.14.1
-  - cartopy
+  - xarray
+  - cartopy<0.20
   - esmpy
-  - intake-xarray
   - geopy
-  - xesmf
+  - xesmf<0.5.0
```

Comment:
```suggestion
  - xesmf
```

Reply:
I agree with all these changes , specially with `xesmf` since oceanspy can already deal with the changes to regridder.